rockcraft/extensions/extension.py

```diff
@@ -87,8 +87,8 @@ def validate(self, extension_name: str) -> None:
             "ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS"
         ):
             raise errors.ExtensionError(
-                f"Extension is experimental: {extension_name!r}",
-                docs_url="https://rockcraft.io/docs/supported-extensions",
+                f"Extension is experimental: {extension_name!r}, please run with ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS=True",
```

Comment:
we do not say please in error messages
https://discourse.ubuntu.com/t/tone-of-voice/18883
this also needs to be in resolution, not the main message

Reply:
removed "please", thanks for the guidelines :). May I know what you are referring to by the main message please?

Reply:
errors have a separate `resolution` field. Together with `doc_slug`, the error could be something like:

```python
raise errors.ExtensionError(
    f"Extension is experimental: {extension_name!r}",
    doc_slug="/reference/extensions/",
    resolution="Run with ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS=True to enable experimental extensions"
)

Reply:
Updated, thank you :)